client/src/context/useAuthContext.tsx

```diff
@@ -43,6 +43,7 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
   // use our cookies to check if we can login straight away
   useEffect(() => {
     const checkLoginWithCookies = async () => {
+      /*
```

Comment:
This seems like a temporary/testing change. Can we revert/untrack these changes?